packages/app/src/cli/prompts/generate/extension.test.ts

```diff
@@ -34,34 +32,34 @@ describe('extension prompt', async () => {
   }
 
   test('when name is not passed', async () => {
-    const answers = {name: 'ext', extensionType: 'ui_extension'}
+    const answers = {name: 'ext', extensionType: 'checkout_ui_extension'}
```

Comment:
why did you change this type, isn't ui_extension also a valid template id?
not saying is wrong, just curious